docs/documentation.md

```diff
@@ -46,14 +46,41 @@ promregator:
   authenticator:
     type: OAuth2XSUAA
     oauth2xsuaa:
-      tokenServiceURL: https://jwt.token.server.example.org/oauth/token
+      url: https://jwt.token.server.example.org
       client_id: myOAuth2ClientId
-    
+
   targets:
     - orgName: myCfOrgName
       spaceName: mySpaceName
       applicationName: myApplication1
-      
+
+    - orgName: myOtherCfOrgName
+      spaceName: myOtherSpaceName
+      applicationName: myOtherApplication
+```
+
+Here is an dummy example for such a configuration yaml file when certificate based authentication is used:
+```yaml
+cf:
+  api_host: api.cf.example.org
+  username: myCFUserName
+  proxy:
+    host: 192.168.111.1
+    port: 8080
+
+promregator:
+  authenticator:
+    type: OAuth2XSUAA
+    oauth2xsuaa:
+      cert_url: https://jwt.cert.token.server.example.org
+      client_id: myOAuth2ClientId
+      client_certificates: "-----BEGIN CERTIFICATE-----\nMyIFu...IxZ\n-----END CERTIFICATE-----\n"
```

Comment:
For the basic auth example the `client_secret` is missing. Hence here is missing the `client_key` ...

Reply:
Yeah, `client_secret` is missing here,  as we suggest to inject it through an environment variable.
Perhaps make this more prominent here (in both cases) by providing a YAML comment?

Reply:
done

Reply:
Something additional on this line. What about the idea of making the example like this:
```suggestion
      client_certificates: |
          -----BEGIN CERTIFICATE-----
          MyIFu...IxZ
          -----END CERTIFICATE-----
```